### PR TITLE
chore: bump start-stack to 2024.01.01

### PR DESCRIPTION
**Pull request by jnussbaum**

### CODE REVIEW

File: src/dsp_tools/commands/xmlupload/ontology_client.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def _get_ontology_names_from_server(self) -> None:
             onto_iris: list[str] = res["project"]["ontologies"]
         except KeyError as e:
             raise BaseError(f"Unexpected response from server: {res}") from e
-        onto_names: list[str] = [iri.split("/")[-1] for iri in onto_iris]
+        onto_names: list[str] = [iri.split("/")[-2] for iri in onto_iris]
```
Comment (jnussbaum) [inline src/dsp_tools/commands/xmlupload/ontology_client.py:63]: The format how DSP-API returns ontology IRIs has changed (see below)

File: src/dsp_tools/resources/start-stack/docker-compose.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ services:
   app:
     # on the verge of every deployment (fortnightly), update the "image" value from the "app" value of
     # https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json
-    image: daschswiss/dsp-app:v11.1.5
+    image: daschswiss/dsp-app:v11.2.0
```
Comment (Nora-Olivia-Ammann) [inline src/dsp_tools/resources/start-stack/docker-compose.yml:10]: This is `"2024.01.02": "app": "v11.3.0",`

**Review by Nora-Olivia-Ammann — CHANGES_REQUESTED** [review]

File: test/unittests/commands/xmlupload/test_ontology_client.py [hunk 3]
```diff
@@ -176,10 +176,10 @@ def test_get_ontology_names_from_server() -> None:
...
             "longname": "Bernoulli-Euler Online",
             "ontologies": [
-                "http://www.knora.org/ontology/0801/biblio",
-                "http://www.knora.org/ontology/0801/newton",
-                "http://www.knora.org/ontology/0801/leibniz",
-                "http://www.knora.org/ontology/0801/beol",
+                "http://api.dasch.swiss/ontology/0801/biblio/v2",
+                "http://api.dasch.swiss/ontology/0801/newton/v2",
+                "http://api.dasch.swiss/ontology/0801/leibniz/v2",
+                "http://api.dasch.swiss/ontology/0801/beol/v2",
```
Comment (Nora-Olivia-Ammann) [inline test/unittests/commands/xmlupload/test_ontology_client.py:182]: When I send the request to a real API from a project I get the ontology as follows:

```
    "ontologies": [
            "http://www.knora.org/ontology/002A/church"
        ],
```

Comment (jnussbaum) [reply test/unittests/commands/xmlupload/test_ontology_client.py:182]: It is a breaking change in DSP-API, so it depends on what server you use.

The following servers still have the old DSP-API version:
 - `dsp-tools start-stack` (on main branch)
 - prod
 - all RDU test servers

And the following servers have the new DSP-API version:
 - most of the dev servers
 - test.dasch.swiss 
 - stage.dasch.swiss

**Review by Nora-Olivia-Ammann — APPROVED** [review]

**Pull request merged**